docs/themes/oso-webpack/index.js

```diff
@@ -370,3 +370,84 @@ window.recordFeedback = (isUp) => {
     }
   });
 }
+
+// Support for code tab groups -- hacks ahead!
+window.addEventListener("load", () => {
+  const unique = (array) =>
+    array.filter((value, index, self) => self.indexOf(value) === index);
+
+  const tabGroupDivs = Array.from(
+    document.querySelectorAll("div[data-tabgroup]")
+  );
+
+  let tabGroups = tabGroupDivs.map(
+    (div) => div.attributes["data-tabgroup"].value
+  );
+  tabGroups = unique(tabGroups);
+
+  for (const tabGroup of tabGroups) {
+    console.log(tabGroup);
+    const codeBlocks = document.querySelectorAll(
+      `div.code[data-tabgroup='${tabGroup}'`
+    );
+    const first = codeBlocks[0];
+    // tabGroupContainer is the full containing div, and replaces the first code
+    // block in the tabGroup
+    const tabGroupContainer = document.createElement("div");
+    tabGroupContainer.className = "code";
+    first.parentNode.insertBefore(tabGroupContainer, first);
+    // tabContainer contains the clickable tabs
+    const tabContainer = document.createElement("div");
+    // codeContainer contains the, well, code
+    const codeContainer = document.createElement("div");
+    tabGroupContainer.appendChild(tabContainer);
+    tabGroupContainer.appendChild(codeContainer);
+    tabContainer.className =
+      "filename rounded-t-md bg-gray-200 text-gray-700 text-sm flex";
```

Comment:
nit: might be more maintainable to create a new class that applies these classes so that (A) more CSS stays in the CSS files and (B) you can tweak the CSS w/o having to re-run the Webpack build process

def not worth spending a lot of time on

Reply:
Oh yeah I actually forgot you can do that. That's also good because it will prevent these CSS classes from being purged